### ISSUE TITLE
CodeJunction community development

### ISSUE THREAD

**Issue by edniltonmatos** [vincent0426/app-ideas#6]
### What are the problems I encountered:
- I would like to share my projects with everyone, but it's unfeasible for everyone to add their challenge examples within the readme.
- I want to see more challenge examples, evaluate them, and give feedback.

### What solution do I see to these problems:
Create a website where users can post their challenge resolution examples, interact with solution posts, host live streams solving the challenge, etc.

### Simpler alternatives:
Discord community.
Facebook community.

**Comment by aldairandre** [comment]
It sounds like a good idea. Probably creating a website might be an expensive thing to do right now, so I think for now creating a Discord community and a Facebook community could be better. What's your opinion man @vicent0426

**Comment by edniltonmatos** [comment]
Some examples of community in this sense:

[Codelandia](https://discord.gg/QevDJqCzaY)
[Umpontoseis](https://discord.gg/p9gdRtm)

**Comment by vincent0426** [comment]
Exactly, I definitely agree with creating a website. But as you may know, creating such a social media platform may require time and resources. Maybe we can create a discord community first.

**Comment by vincent0426** [comment]
> Discord channel [link](https://discord.gg/vBBRu5jr)
> 
> It still needs some changes so if you want to help, call me and I'll put you as a moderator

link is broken

**Comment by edniltonmatos** [comment]
> > Discord channel [link](https://discord.gg/vBBRu5jr)
> > 
> > It still needs some changes so if you want to help, call me and I'll put you as a moderator
> 
> link is broken

try again

**Comment by edniltonmatos** [comment]
I tested it with other people and they were able to access it with this link. Wonder what's going on? :thinking:

**Comment by aldairandre** [comment]
@vincent0426 i managed to join the discord group through the link.
@edniltonmatos  The link it's working.
